misc/git/hooks/pylint

```diff
@@ -25,7 +25,9 @@ function msg() {
   echo "$(date) $@"
 }
 
-PYLINT=/usr/bin/gpylint
+if [[ -z "$PYLINT" ]]; then
+  PYLINT=/usr/bin/gpylint
+fi
```

Comment:
There is a bash syntax:
PYLINT=${PYLINT:-/usr/bin/gpylint}
not sure it's simpler.

Reply:
Thanks! I didn't think about that.

I find it less explicit but more Bash idiomatic :) Therefore, I've changed it to this version.